content/rest/channels-messages.textile

```diff
@@ -209,6 +224,12 @@ if result_page.has_next?
   puts "#{next_page.items.length} messages received in second page"
 end
 
+bc[python]. result_page = channel.history()
+print str(len(result_page.items)) + " messages received in first page"
```

Comment:
it is ok in Python to use double quotes for strings that will be outputted and single quotes for internal strings: http://programmers.stackexchange.com/a/155188/80170